README.md

```diff
@@ -14,8 +14,9 @@ The project structure is based on [`drupal-composer/drupal-project`](https://git
 1. Clone this repository
 2. From the root of the cloned repository:
   1. Run `composer install`
-	2. Run `docker-compose build`
-	3. Run `docker-compose up`
+  2. Copy `example.docker-compose.override.yml` to `docker-compose.override.yml` and fill out the required information.
```

Comment:
We could add two sections under Installation instructions: Required steps and Optional steps.